src/sentry/event_manager.py

```diff
@@ -1107,6 +1111,10 @@ def _save_aggregate(event, flat_hashes, hierarchical_hashes, release, **kwargs):
         try:
             short_id = project.next_short_id()
         except OperationalError:
+            metrics.incr(
+                "next_short_id.timeout",
+                tags={"platform": event.platform or "unknown"},
```

Comment:
Do we also want to add a tag that tells us if we're emitting it from _save_aggregate or _save_aggregate2, or it shouldn't matter?

Reply:
We are now 100% using the `_save_aggregate2` and I don't expect us to go back to the old implementation since no negative effects of the new transaction use were observed.